internal/db/service/group.go

```diff
@@ -0,0 +1,224 @@
+package service
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	snitchv1 "snitch/pkg/proto/gen/snitch/v1"
+
+	"connectrpc.com/connect"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+// CreateGroupDatabase ensures the group database exists and tables are created
+func (s *DatabaseService) CreateGroupDatabase(
```

Comment:
Maybe we should refactor these methods into more of a repo fashion? It seems weird to have group methods and repo methods share the same logical structure